crates/app/tests/mod.rs

```diff
@@ -78,10 +87,16 @@ async fn app() {
         &tag,
         &"/".parse().unwrap(),
         mime::TEXT_PLAIN,
-        b"test".to_vec(),
+        TEST_DATA.to_vec(),
     )
     .await;
 
+    let (test_resp, test_type) =
+        tree::get_path(&cl, &addr, &foo, &tag, &"/".parse().unwrap()).await;
+
+    assert_eq!(test_type, mime::TEXT_PLAIN);
+    assert_eq!(test_resp, TEST_DATA.to_vec());
```

Comment:
probably a nit, but I still think we should use a string literal directly, the constant obfuscates the behavior here and is not consistent with existing codebase.
Just use `"test"`

Reply:
I disagree, it's probably safer and cleaner to use a variable for a string which gets used in more than one place. But I'll comply.